### ISSUE TITLE
Quick start guide doesn't produces .html file

### ISSUE THREAD

**Issue by ronnross** [kss-node/kss-node#217]
While going through the [Quick Start Guide](https://github.com/kss-node/kss-node/wiki/Quick-Start-Guide) an html file is never produced. I have followed the tutorial up to and including the "Running kss-node" step. There is a styleguide folder creating and within there is a public folder. There is .html file therefore I'm unable to view the generated style guid. The guild leads me to believe something that I can view should have been produced. Let me know if you need more information.

**Comment by lukeapage** [comment]
+1
```
kss init sgt
kss source/less styleguide --template sgt
```

gives the error
```
{ [Error: ENOENT, no such file or directory 'path\to\sgt\index.html']
  errno: -4058,
  code: 'ENOENT',
  path: 'path\\to\\sgt\\index.html',
  syscall: 'open' }
```

You have to go to the template source directory and copy the files not in the public folder.

**Comment by JohnAlbin** [comment]
The `init` options is legacy option that was needed for 1.x. I never use it with 2.x.

> You have to go to the template source directory and copy the files not in the public folder.

Again, don't create a custom template. That's not good advice anymore. You can hotlink your CSS files into your generated style guide with the `--css` option (as mentioned in the quick start guide.)

Getting back to @ronnross's request:

I followed the quick start guide and ran `npm init` and the kss install commands inside a `test/` folder. And then created the sample CSS (with KSS comments) and put it into test/styles/test.css. Then I ran the command in the quick start guide and saw this output:
```
$ npm-exec kss-node --source styles/ --destination styleguide --css styles/test.css

Generating your KSS style guide!

 * KSS Source  : /Users/jwilkins/Sites/test/styles
 * Destination : /Users/jwilkins/Sites/test/styleguide
 * Template    : /Users/jwilkins/Sites/test/node_modules/kss/generator/handlebars/template
 * Helpers     : 

...Parsing your style guide:
 - /Users/jwilkins/Sites/test/styles/test.css
...Determining section markup:
...Generating style guide sections:
 - section base [ Unnamed ]
 - homepage
   ...no homepage content found in styleguide.md.

Generation completed successfully!

$ ls -l styleguide/
total 24
-rw-rw-r--   1 jwilkins  staff  1789 Jul 29 01:35 index.html
drwxrwxr-x  13 jwilkins  staff   442 Jul 29 01:35 public
-rw-rw-r--   1 jwilkins  staff  1952 Jul 29 01:25 section-base.html
-rw-rw-r--   1 jwilkins  staff  2214 Jul 29 01:35 section-hard-rule.html
```

It generated the style guide just fine. Perhaps you could explain the exact steps you took?

**Comment by lukeapage** [comment]
> The init options is legacy option that was needed for 1.x. I never use it with 2.x.

I agree you don't need it. But in our case we wanted to restyle the styleguide and use the same elements of our main site so the styleguide matched the style we were trying to show.. if that makes sense. So I wouldn't categorise it legacy if I were you, just only needed for advance use cases.

I just re-ran a test and realised what I did wrong, I used `init` instead of `--init` :S
